airbyte_cdk/sources/declarative/interpolation/macros.py

```diff
@@ -177,6 +177,7 @@ def format_datetime(
         dt_datetime = (
             datetime.datetime.strptime(dt, input_format) if input_format else str_to_datetime(dt)
         )
+    dt_datetime = dt_datetime.replace(tzinfo=pytz.utc)
```

Comment:
Baz mentioned that this should only happen if we don't have the timezone information. We should be able to do that by checking if the resulting datetime is naive. I'll update that later today